test.sh

```diff
@@ -15,7 +15,7 @@
 
 COLORS_RED='\033[0;31m'
 COLORS_RESET='\033[0m'
-LOG_LEVEL="fatal"
+LOG_LEVEL="error"
```

Comment:
it makes test.sh loud af, cuz there's lots of tests which are expected to fail, I'd just add `--verbose` flag to the CI.

Reply:
This was just a temporary change, so that I could see what was wrong because it was impossible to determine the origin of the issue from the generic "service exited with error" message.